x-pack/plugins/triggers_actions_ui/public/application/lib/monitoring_utils.ts

```diff
@@ -16,10 +16,21 @@ export function getFormattedDuration(value: number) {
   if (!value) {
     return '00:00';
   }
+
   const duration = moment.duration(value);
-  const minutes = Math.floor(duration.asMinutes()).toString().padStart(2, '0');
-  const seconds = duration.seconds().toString().padStart(2, '0');
-  return `${minutes}:${seconds}`;
+  let minutes = Math.floor(duration.asMinutes());
+  let seconds = duration.seconds();
+  const ms = duration.milliseconds();
+
+  if (ms >= 500) {
```

Comment:
Is there nothing in moment that does this logic for us? Or is this something custom we want?

Reply:
yea I looked into it, it doesn't seem like moment#durations offer this functionality https://momentjs.com/docs/#/durations/

especially since we want to format something as mm:ss where the minutes can be arbitrarily large (i.e 900:00 for 900 minutes)

Reply:
Sounds good!